NAMESPACE

```diff
@@ -5,6 +5,7 @@ export(read_epp_data)
 export(read_epp_subpops)
 export(read_hiv_pop)
 export(read_hivproj_param)
+export(read_spectrum_region_code)
```

Comment:
small thing, but should this be simply `read_region_code` or is there something spectrum specific about the code? Otherwise the whole package is about reading spectrum data so it feels like it could be omitted (looking at the other exports)

Reply:
Ah yeah it is a specific spectrum region code but I agree - this whole package is spectrum.
Will update that.